connectors/connectors.go

```diff
@@ -730,3 +753,118 @@ func StartPeriodic(ctx context.Context, t time.Duration, fn func()) {
 	/* call handler immediately */
 	go handler()
 }
+
+// TODO: move methods below from connectors.go to separated 'utils' file?
```

Comment:
That's the best way I've come up with to parse duration (exactly duration, not date or time of the day where we can use standard formatters).
3 minutes / 73 hours (less than 73 hours, 72 hours = 3 days, i.e. less than 3 days) were taken intuitively, how it feels to me it will be more clear for end-user: 
for example "150 seconds" seems more informative as "150 seconds" than "2 minutes"
but "500 seconds" is more informative as "8 minutes" than "500 seconds"
same for hours and days, but we can change these limits.
Commented this in commit https://github.com/gwos/tcg/pull/208/commits/49938cefc12ce7ebb094abea263966250e3c9b12